linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/tests/pam_tally2_auth_unlock_time_configured.pass.sh

```diff
@@ -4,15 +4,28 @@
 cat >/etc/pam.d/common-account <<CAPTC
 account	[success=1 new_authtok_reqd=done default=ignore]	pam_unix.so
 account	requisite			pam_deny.so
-account required                        pam_tally2.so
+account required            pam_tally2.so
```

Comment:
Is it intentional? It seems more readable as it was before.

Reply:
Besides the need to rebase, I have this pending comment. Then it should be ready.